lib/build/node-sass.js

```diff
@@ -139,10 +139,18 @@ module.exports = function (bannerWidth, libraryPaths) {
 
     /**
      * Handler for successful transpilation using node-sass.
-     * @param {string} css Compiled css
+     * This functions gets called with an object containing a CSS file and its sourcemap,
+     * which is modified and passed through CSS rework, before being pushed to the results.
+     *
+     * @param {object} results Results object from node-sass, which contains:
+     *   - css {string}
+     *   - map {string}
+     *   - stats {object}
      * @param {string} map The source-map for the compiled css
```

Comment:
You are looking for **record type**.
Per the [specification](http://usejsdoc.org/tags-type.html) I believe this `@param` would be written as follows:

 `@param {{css:string, map:string, stats:object}} results Results object from node-sass`